homeworkcss.css

```diff
@@ -0,0 +1,51 @@
+.picture {
+    width: 100%;
+    height: 550px;
+    margin-top: 40px;
+}
+    
+header {
+    background-color: rgb(165, 42, 42);
+    font-size: 20px;
+    font-weight: bold;
+    color: white;
+    text-align: center;
+    padding: 5px;
+    margin-top: 20px;
+}
+
+p {
+    font-size: 18px;
+}
+.LOREMSUKA {
+    font-size: 18px;
+    margin-top:100px;
+}
+.copyri {
+    color: white;
+    background-color: rgb(0, 0, 0);
+    text-align: center;
+}
+
+.car1 {
+    float: left;
+    border: 2px dashed rgb(0, 0, 0);
+    margin-left: 40%;
+}
+.car2 {
+    float: left;
+    border: 2px dashed rgb(0, 0, 0);
+}
+.car3 {
+    float: left;
+    border: 2px dashed rgb(0, 0, 0);
+}
+.car4 {
+    float: left;
+    border: 2px dashed rgb(0, 0, 0);
+}
+.car5 {
+    float: left;
+    border: 2px dashed rgb(0, 0, 0);
+    margin-right: 40%;
+}
```

Comment:
All classes have some properties repeated, use a common class (car) for all, and set indents to the wrapper (car-img-wrapper) for example